packages/gpu-operator/v24.3.0+1/package.yaml

```diff
@@ -0,0 +1,20 @@
+# yaml-language-server: $schema=https://glasskube.dev/schemas/v1/package-manifest.json
+
+name: "gpu-operator"
+shortDescription: >-
+  The GPU Operator allows administrators of Kubernetes clusters to manage GPU nodes just like CPU nodes in the cluster.
```

Comment:
![image](https://github.com/user-attachments/assets/b86cbcbe-2816-4cea-8d61-57eebdcad3ef)

I think the short description is a little too long (the interesting part is truncated; see screenshot). Maybe something like this would fit better?

> Management of NVIDIA components needed to provision GPU